src/components/Table/Pagination.jsx

```diff
@@ -26,9 +28,13 @@ const SizedPagination = sizeMe({ noPlaceholder: true })(
 
 SizedPagination.propTypes = {
   isItemPerPageHidden: PropTypes.bool,
+  // It is currently not possible to completely disable the carbon pagination
+  // therefor we use this prop to combine disable and hiding of controls.
+  preventInteraction: PropTypes.bool,
```

Comment:
Why does the pagination select need to be hidden? The disable via [`pageInputDisable` prop](https://github.com/carbon-design-system/carbon/blob/468d6ac9a524b031f920bcb8048b8c4e3e865823/packages/react/src/components/Pagination/Pagination.js#L126-L129) should be enough imo

Reply:
See comment above. I discussed with Taylor a few options how to handle this, including trying to disable the control via CSS and JS, but we came to the conclusion that hiding it would be the cleanest and safest way.

Reply:
is there a defect opened back to the carbon site to remedy this?